website/build_site.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+cp -r ../documentation content/
+jupyter nbconvert ../source/Mlos.Notebooks/*.ipynb --to markdown --output-dir content/notebooks
+
+cp ../*.md content/
+cp ../LICENSE content/
+cp ../README.md content/_index.md
+mv content/documentation/README.md content/documentation/_index.md
+sed -i 's/.md/\//g' *.md
```

Comment:
not the most specific regex in the world, but it works? hm should the ``.`` be escaped? I'm no good with sed